File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/switch-case/code-examples/switch-two-nested.js
const status = 'draft';
const format = 'text';

const channel = 'web';

switch (status) {
    case 'draft':
    case 'review':
    case 'revision':
        prepareEditing();
        switch (format) {
            case 'text':
                openTextEditor();
                break;
            case 'image':
                openImageEditor();
                break;
            default:
                openPlainEditor();
                break;
        }
        break; // Exit the outer switch after the inner switch.
    case 'published':
        prepareDelivery();
        switch (channel) {
            case 'web':
                publishOnline();
                break;
            case 'print':
                queuePrintJob();
                break;
            default:
                queueManualDelivery();
                break;
        }
        break;
    default:
        showStatusHint();
        break;
}

// Continue after the selected action sequence.
continueProcess();

File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/switch-case/code-examples/switch-two-nested.cs
string status = "draft";
string format = "text";

string channel = "web";

switch (status) {
    case "draft":
    case "review":
    case "revision":
        PrepareEditing();
        switch (format) {
            case "text":
                OpenTextEditor();
                break;
            case "image":
                OpenImageEditor();
                break;
            default:
                OpenPlainEditor();
                break;
        }
        break; // Exit the outer switch after the inner switch.
    case "published":
        PrepareDelivery();
        switch (channel) {
            case "web":
                PublishOnline();
                break;
            case "print":
                QueuePrintJob();
                break;
            default:
                QueueManualDelivery();
                break;
        }
        break;
    default:
        ShowStatusHint();
        break;
}

// Continue after the selected action sequence.
ContinueProcess();
